app/lib/common/widgets/acter_icon_picker/model/acter_icons.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/cupertino.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+
+enum ActerIcons {
```

Comment:
This contains set of ActorIcons data.

![Screenshot 2024-09-05 at 1 55 06 PM](https://github.com/user-attachments/assets/bea3f45d-5c34-4363-aa35-8db4d90869f3)